DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata.py

```diff
@@ -508,10 +539,14 @@ def __findSubdirMissingMeta(self, meta, pathSelection):
   def __expandMetaDictionary(self, metaDict, credDict):
     """ Expand the dictionary with metadata query
     """
-    result = self.getMetadataFields(credDict)
+    result = self.getMetadataFields(credDict) # this now returns a fully qualified meta names, unlike the query !
```

Comment:
Again this is a comment reflecting this particular release event, not relevant in the long term